docs/instance.md

```diff
@@ -48,32 +48,30 @@ and creates the proper databases ($PREFIX/$DOCTYPE) for these doctypes:
 - `io.cozy.apps`
 - `io.cozy.manifests`
 - `io.cozy.files`
-- `io.cozy.folders`
 - `io.cozy.notifications`
 - `io.cozy.settings`
 
 Then, it creates the following indexes for these doctypes :
 
 - `io.cozy.apps : ["slug"]`
 - `io.cozy.manifest : ["url"]`
-- `io.cozy.files & io.cozy.folders: ["FolderId", "name"], ["FolderId", "modifiedDate"]`
```

Comment:
should be ["dir_id", "updated_at"] or let's just remove it as we dont create it for now !